packages/main/src/FileUploader.js

```diff
@@ -427,7 +427,7 @@ class FileUploader extends UI5Element {
 	}
 
 	get _canUseNativeFormSupport() {
-		return !!this.attachInternals;
+		return this._internals && this.internals.setFormValue;
```

Comment:
`this._internals.setFormValue` (with underscore)

Reply:
Done